pkg/awx/connection.go

```diff
@@ -137,11 +147,27 @@ func (b *ConnectionBuilder) Build() (c *Connection, err error) {
 		return
 	}
 
+	if b.cacrt != nil && len(b.cacrt) > 0 && b.insecure {
+		err = fmt.Errorf("CA certificate and insecure are mutually exclusive")
+		return
+	}
+	var cert_store *x509.CertPool
+	if b.cacrt != nil && len(b.cacrt) == 0 {
+		cert_store, err = x509.SystemCertPool()
```

Comment:
Question: Do we want to use the system certificate pool *and* `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt` by default, or just the system pool without appending the kubernetes ca.crt?

Reply:
I think that only the system certificate pool, as the AWX server certificate won't probably be verifiable with the Kubernetes CA.